source/MRMesh/MRIdentifyVertices.cpp

```diff
@@ -8,6 +8,13 @@ namespace MR
 namespace MeshBuilder
 {
 
+void unsignZeroValues( Vector3f& v )
```

Comment:
probably put it in Vector3, or make it a general algorithm that can be applied to Vector2/3/4?